config/samples/example-test.sh

```diff
@@ -486,6 +511,31 @@ kubectl apply -f config/samples/core_v1alpha1_nginx_componentplan.yaml --dry-run
 waitComponentPlanRetryTime "default" "do-once-nginx-sample-15.0.2" "5"
 info "5.7.4 verify this componentplan will failed, show error log"
 kubectl get cpl do-once-nginx-sample-15.0.2 --output='jsonpath={.status.conditions[?(@.type=="Actioned")]}'
+info "" # go to a new line
```

Comment:
These parts(basic auth) seem to exist upstream and may need to be rebaseed.

Reply:
When I run the test locally, I realized that some title did not go to a new line. Therefore I added a new line there so that the output looks better. Can I know if this change should be removed?

Reply:
mainly about 5.8 helm basic auth part.